README.md

```diff
@@ -9,95 +9,97 @@ Minecraft uses resource packs to assign characters (glyphs) to different unicode
+|        Right Stick Button        |  Windows MR  |          |  U+E0C4 |                      ![](./images/mr-rs.png)                     |
+|       Left Touchpad Button       |  Windows MR  |          |  U+E0C5 |                ![](./images/mr-left-touchpad.png)                |
+|  Left Touchpad Horizontal Button |  Windows MR  |          |  U+E0C6 |           ![](./images/mr-left-touchpad-horizontal.png)          |
+|   Left Touchpad Vertical Button  |  Windows MR  |          |  U+E0C7 |            ![](./images/mr-left-touchpad-vertical.png)           |
+|       Right Touchpad Button      |  Windows MR  |          |  U+E0C8 |                ![](./images/mr-right-touchpad.png)               |
+| Right Touchpad Horizontal Button |  Windows MR  |          |  U+E0C9 |          ![](./images/mr-right-touchpad-horizontal.png)          |
+|  Right Touchpad Vertical Button  |  Windows MR  |          |  U+E0CA |           ![](./images/mr-right-touchpad-vertical.png)           |
+|        Left Trigger Button       |  Windows MR  |          |  U+E0CB |                      ![](./images/mr-lt.png)                     |
+|       Right Trigger Button       |  Windows MR  |          |  U+E0CC |                      ![](./images/mr-rt.png)                     |
+|              Windows             |  Windows MR  |          |  U+E0CD |                   ![](./images/mr-windows.png)                   |
+|             0 Button             |     Rift     |          |  U+E0E0 |                    ![](./images/rift-zero.png)                   |
+|             A Button             |     Rift     |          |  U+E0E1 |                     ![](./images/rift-a.png)                     |
+|             B Button             |     Rift     |          |  U+E0E2 |                     ![](./images/rift-b.png)                     |
+|         Left Grab Button         |     Rift     |          |  U+E0E3 |                      ![](images/rift-lg.png)                     |
+|         Right Grab Button        |     Rift     |          |  U+E0E4 |                      ![](images/rift-rg.png)                     |
+|         Left Stick Button        |     Rift     |          |  U+E0E5 |                     ![](./images/rift-ls.png)                    |
+|        Right Stick Button        |     Rift     |          |  U+E0E6 |                     ![](./images/rift-rs.png)                    |
+|        Left Trigger Button       |     Rift     |          |  U+E0E7 |                     ![](./images/rift-lt.png)                    |
+|       Right Trigger Button       |     Rift     |          |  U+E0E8 |                     ![](./images/rift-rt.png)                    |
+|             X Button             |     Rift     |          |  U+E0E9 |                     ![](./images/rift-x.png)                     |
+|             Y Button             |     Rift     |          |  U+E0EA |                     ![](./images/rift-y.png)                     |
```

Comment:
Bit big of a change, eh? Looks like alot useless additional spaces all of the sudden

Reply:
I didn't want to disturb the flow of how nice it looks in code view. :D 
![image](https://user-images.githubusercontent.com/10010369/93646773-c926e400-f9fe-11ea-8c1c-d00d9c00639a.png)

Reply:
![image](https://user-images.githubusercontent.com/8733998/93654841-30a15b80-fa20-11ea-9993-55087d23d007.png)

Reply:
Yeh, it was like that when I found it. I've just added 18 spaces to each side anyway. With the text editor I use, I can click multiple lines, so I added a ton of  extra spaces to make the rest fit in. 

Original (before I edited it): 
![image](https://user-images.githubusercontent.com/10010369/93655049-144df080-fa19-11ea-8eb9-95b9a91d5047.png)

Reply:
Before
![image](https://user-images.githubusercontent.com/8733998/93655454-b246b880-fa23-11ea-9b25-6bd77d7e12b7.png)
After
![image](https://user-images.githubusercontent.com/8733998/93655457-becb1100-fa23-11ea-96dd-c53286f795da.png)
Added visual effect on the outcome: none
Added loadtime: 53ms to 93ms -> 40ms
Added spaces: 1504
Added noise in git: 94 lines

And why?

Obviousely trolling here, have a good night.